neqo-transport/src/server.rs

```diff
@@ -445,16 +448,23 @@ impl Server {
             connections: self.connections.clone(),
         }));
 
+        let draft_version = if quic_version == 0xff00_0000 + 27 {
+            DraftVersion::Draft27
+        } else {
+            DraftVersion::Draft28
+        };
```

Comment:
Can't you use DraftVersion::from() or try_from()?

Reply:
:+1: